cognee/api/v1/search/search_v2.py

```diff
@@ -14,11 +14,13 @@
 from cognee.tasks.chunks import query_chunks
 from cognee.tasks.graph import query_graph_connections
 from cognee.tasks.summarization import query_summaries
+from cognee.tasks.compute import query_compute
 
 class SearchType(Enum):
     SUMMARIES = "SUMMARIES"
     INSIGHTS = "INSIGHTS"
     CHUNKS = "CHUNKS"
+    COMPUTE = "COMPUTE"
```

Comment:
I meant `completion` like it usually exists in LLM APIs -> https://llm-engine.scale.com/guides/completions/

Reply:
Will make the change